driver/k8s/logical_volume.go

```diff
@@ -165,27 +127,23 @@ func (s *logicalVolumeService) CreateVolume(ctx context.Context, node string, na
 
 func (s *logicalVolumeService) DeleteVolume(ctx context.Context, volumeID string) error {
 	lvList := new(topolvmv1.LogicalVolumeList)
-	err := s.mgr.GetClient().List(ctx, lvList,
-		client.MatchingField(indexFieldVolumeID, volumeID))
+	err := s.List(ctx, lvList, client.MatchingField(indexFieldVolumeID, volumeID))
```

Comment:
```suggestion
	err := s.List(ctx, lvList, client.MatchingFields{indexFieldVolumeID: volumeID})
```

`client.MatchingField` is deprecated.